fiaas_skipper/deploy/deploy.py

```diff
@@ -31,12 +31,14 @@ def __init__(self, cluster, release_channel_factory, bootstrap, spec_config=None
         else:
             self._spec_config = spec_config
 
-    def deploy(self):
+    def deploy(self, namespaces=None):
```

Comment:
You can default this to `tuple()`, and then the if check can be simply `if deployment_config.namespace not in namespaces`.

Reply:
Ok will do.

Reply:
I'm fixing this now, and writing a test to cover it. I'll send a new PR later today.